tests/component.test.js

```diff
@@ -60,20 +60,28 @@ describe('Component', () => {
 
   test('When lazy is set to true, should only emit input onChange', async () => {
     // default settings
-    const wrapper = createWrapper({ lazy: true })
+    const wrapper = createWrapper({
+      props: { modelModifiers: { lazy: true } },
+      attributes: {
+        'update:modelValue'(value) {
+          wrapper.setProps({ modelValue: value })
+        }
+      }
+    })
+
     const input = wrapper.find('input')
 
     input.trigger('input')
-    expect(wrapper.emitted().input).toBeFalsy()
+    expect(wrapper.emitted().input.length).toBe(1)
 
     input.trigger('change')
-    expect(wrapper.emitted().input).toBeTruthy()
+    expect(wrapper.emitted().input.length).toBe(1)
     expect(wrapper.emitted().change).toBeTruthy()
   })
 
   test('When lazy is set to false, should not emit input on change', async () => {
     // default settings
-    const wrapper = createWrapper({ lazy: false })
+    const wrapper = createWrapper({ props: { lazy: false } })
```

Comment:
False positive since lazy prop was removed right? Can probably remove this test altogether.

- [ ] Remove test

Reply:
ah good catch, I updated the test above for the `modelModifiers` but missed this one. will update